pilot/lib/pilot/options.rb

```diff
@@ -42,7 +42,23 @@ def self.available_options
                                      optional: true,
                                      code_gen_sensitive: true,
                                      default_value: ENV["TESTFLIGHT_APPLE_ID"],
-                                     default_value_dynamic: true),
+                                     default_value_dynamic: true,
+                                     type: String,
+                                     verify_block: proc do |value|
+                                       error_message = "`apple_id` value is incorrect. App ID consists of a Team ID and a bundle ID search string, with a period (.) separating the two parts. You can find your Team ID here: https://developer.apple.com/account/#/membership and your bundle ID in Xcode settings."
```

Comment:
👋 @janpio , 

I really like your error / documentation / message suggestions 🙌 Have a feeling this one might be a bit too much technical though 🤔 Thoughts how to improve it? 🙏